src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,44 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
```

Comment:
The `alive` array is defined as a static property of the `Animal` class. This is correct, but ensure that it is managed properly when animals are removed or added.